apps/test-viewer/src/components/repositories-tree/RepositoriesTree.tsx

```diff
@@ -53,7 +43,7 @@ function Tree({ accessToken, itwinId, noDataMessage, environment }: Omit<Reposit
       );
     }
 
-    if (rootNodes.length === 0 && !isLoading) {
+    if (rootNodes.length === 0 && !treeProps.isLoading) {
```

Comment:
I think `isLoading` should be extracted from `useTree` result (similar to `rootNodes`) so that it doesn't land into the `treeProps` object.